README-it.md

```diff
@@ -0,0 +1,467 @@
+        final CustomIncidentHandlerPlugin plugin = new CustomIncidentHandlerPlugin();
+        plugin.setIntervalMs(intervalMs);
+        plugin.setUrl(url);
+        plugin.setFallbackMailReceiver(fallbackMailReceiver);
+        plugin.setUsername(username);
+        plugin.setPassword(password);
+        plugin.setHost(host);
+        plugin.setPort(port);
+        plugin.setMailSender(mailSender);
+        plugin.setMailBodyTemplate(mailBodyTemplate);
+        plugin.setIncidentTemplate(incidentTemplate);
+        plugin.setSubject(subject);
+        return plugin;
+    }
+}
+```
+
+### Step 5
+
+Avvia l'applicazione. Ora riceverai via email gli errori dei processi.
+
+## Dove posso trovare un esempio del plugin Incident Listener lanciato con Spring Boot?
+
+Puoi usare l'applicazione demo [incident-listener-spring-boot-demo](https://github.com/jit-open/incident-listener-spring-boot-demo/tags). 
+Essa ha due tag:
+
+ * `Initial_Version` è la versione dell'applicazione senza il plugin
+ * `Incident_Listener_Works` è la versione con il plugin incident listener parzialmente impostato. Perchè sia funzionante bisogna
+   rinominare il file `src/main/resources/application.yaml-template` in `application.yaml` e sostituire i `TODO` nel file con i valori effettivi. 
+   Inserisci l'indirizzo email del destinatario in
+   `src/main/resources/sample-process.bpmn` (attività `Set up incident listener`, tab `Input/Output`, pannello `Output`). 
+   Successivamente puoi avviare l'applicazione Spring Boot ed il listener si avvia in automatico con esso.
+
+## Ho un processo con molteplici attività chiamate. Come posso definire il destinatario solo una volta?
+
+Per assicurare che le proprietàd del Incident Listener vengano passate ai sotto processi, aggiungi le mappature nella sezione `Variables`
+del Camunda Modeler. Ci devono essere 2 `In Mappings` per `incidentNotificationReceiver` e 
+`incidentNotificationCc`. In entrambi i casi, devi utilizzare la seguente coinfigurazione:
+
+ * `Type`: `Source`
+ * `Source` e `Target` devono valere il nome delle rispettive variabili (`incidentNotificationReceiver` e
+   `incidentNotificationCc`).
+
+![Come passare le variabili ai sotto processi][subprocess]
+
+Puoi trovare un esempio nel [subprocess_example_parent.bpmn](docs/subprocess_example_parent.bpmn). Puoi anche vederlo in azione 
+nel [esempio con Spring Boot](https://github.com/jit-open/incident-listener-spring-boot-demo/). Per far ciô, 
+
+1. apri [subprocess_example_parent.bpmn](https://github.com/jit-open/incident-listener-spring-boot-demo/blob/master/src/main/resources/subprocess_example_parent.bpmn),
+2. nell'attività `Set up incident listener` cambia la variabile `incidentNotificationReceiver` e `incidentNotificationCc` in indirizzi mail effettivi,
+3. avvia l'applicazione, e
+4. avvia il processo `Subprocess Example (Parent)` nella task list.
+
+## Versioni
+
+ * `1.0.0`: Initial version.
+
+## Maintainer
```

Comment:
Please remove the `Maintainer` section. We are offering this plugin "as-is", i. e. without support.